sapl/sessao/views.py

```diff
@@ -1498,8 +1499,14 @@ def get_materias_ordem_do_dia(sessao_plenaria):
         ementa_observacao = o.observacao
         titulo = o.materia
         numero = o.numero_ordem
-        tramitacao = o.materia.tramitacao_set.last()
-        turno = None
+        
+        tramitacao = ''
+        tramitacoes = Tramitacao.objects.filter(materia=o.materia)
+        for aux_tramitacao in tramitacoes:
```

Comment:
Dá pra fazer assim: `tramitacao = next([ t for t in tramitacoes if t.turno], '')`